src/main/config/config.c

```diff
@@ -413,6 +413,14 @@ void createDefaultConfig(master_t *config)
     config->gyro_sync_denom = 4;
     config->pid_process_denom = 2;
 #endif
+
+// I2c MPU6500 targets cannot do High speed gyro sampling on EXTI so those are limited to 1k 188hz (non working MPU divider)
+#if defined(USE_GYRO_MPU6500) && !defined(USE_GYRO_SPI_MPU6500)
+    config->gyro_lpf = 1;
```

Comment:
Can we get rid of the magic numbers? That is define:
```
enum lpf_e {
    INV_FILTER_256HZ_NOLPF = 0,
    INV_FILTER_188HZ,
    INV_FILTER_98HZ,
    INV_FILTER_42HZ,
    INV_FILTER_20HZ,
    INV_FILTER_10HZ,
    INV_FILTER_5HZ,
    INV_FILTER_2100HZ_NOLPF,
    INV_FILTER_COUNT
};
```
in `accgyro_mpu.h` and then use `INV_FILTER_188HZ`